messing around~

rae-stanton opened the pull request:
just messing around. that's it.
<img src="https://media0.giphy.com/media/H7F0zSaC9ZHZC/giphy.gif"/>

rae-stanton committed:
messing around~

just messing around. that's it.

spaulsandhu reviewed (changes requested):
I challenge you to apply a different class to the badge, based on the difficulty.

rae-stanton committed:
Updated the difficulty styling

This takes the fun gradient upgrade to the recipe's difficulty badge and adds a cleanup for the repetition in the view code in `recipes/show` and `_card` partial. Now it displays different gradients depending on the recipe's level of difficulty.

rae-stanton commented:
It is done. (With cleanup!)
Preview it here:
[Video Here](https://youtu.be/r0lzQAVNPZU)

spaulsandhu reviewed (approved):
Looks great!!!!!
